src/functions-and-arrays.js

```diff
@@ -1,19 +1,73 @@
 // Iteration #1: Find the maximum
 
+const maxOfTwoNumbers = (number1, number2) => {
+  if (number1 > number2) {
+    return number1;
+  } else {
+    return number2;
+  }
+}
+
 // Iteration #2: Find longest word
 const words = ['mystery', 'brother', 'aviator', 'crocodile', 'pearl', 'orchard', 'crackpot'];
 
+function findLongestWord(arrayOfWords) {
+  let word = "";
+  for (let i = 0; i < arrayOfWords.length; i++) {
+    if (word.length < arrayOfWords[i].length) {
+      word = arrayOfWords[i];
+    }
+  }if (!arrayOfWords.length) {
```

Comment:
What about placing this condition above in the function body? This way, we first evaluate whether there are elements to be checked. Only if there are elements, we want to find the longest word.